core/lib/libtomcrypt/mpi_desc.c

```diff
@@ -545,8 +545,7 @@ static int montgomery_reduce(void *a, void *b, void *c)
 	if (mbedtls_mpi_grow(&A, N->n + 1))
 		goto out;
 
-	if (mbedtls_mpi_montred(&A, N, *mm, &T))
-		goto out;
+	mbedtls_mpi_montred(&A, N, *mm, &T);
```

Comment:
This change should be in the commit "Squashed commit upgrading to mbedtls-2.27.0"? Right?

Reply:
NO，Squashed commit upgrading to mbedtls-2.27.0 this patch is from CVE. The update above is not included in the patch.

Reply:
Please check this commit: https://github.com/OP-TEE/optee_os/commit/7901324d9530594155991c8b283023d567741cc7